.github/workflows/continuous-integration.yaml

```diff
@@ -36,7 +36,36 @@ jobs:
         yarn build
         yarn lint
 
+    - name: 🌐 Setup local test network
+      working-directory: packages/e2e
+      run: |
+        yarn local-network:up -d
+      env:
+        POSTGRES_PORT: '5435'
+        OGMIOS_PORT: '1340'
+        OGMIOS_URL: 'ws://cardano-node-ogmios:1340'
+
     - name: 🔬 Test
       run: |
         yarn test:build:verify
         yarn test
+        yarn workspace @cardano-sdk/e2e test:wallet
+      env:
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+        BLOCKFROST_API_KEY: ''
```

Comment:
Is this a required ENV? We should adjust our config validation

Reply:
I think is required now, I wil fix this as part of this PR

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/424/commits/4dbc20b6cd320dbc026cad4a926523cc5db605b4 also see https://github.com/input-output-hk/cardano-js-sdk/pull/424#discussion_r964356053